test-resources/swift/cleanup_rules/configurations/rules.toml

```diff
@@ -36,3 +36,30 @@ replace_node = "parameter_access"
 replace = "true"
 holes = ["stale_flag"]
 groups = ["replace_expression_with_boolean_literal"]
+
+#
+# For @stale_flag_name = stale_flag and @treated = true
+# Before 
+#   !TestEnum.stale_flag.isEnabled
+# After 
+#   false
+#
+
+[[rules]]
+name = "replace_isToggleDisabled_with_boolean_literal"
+query = """(
+(navigation_expression
+        target: (navigation_expression
+            target: (prefix_expression
+                operation: (bang))
+            suffix: (navigation_suffix
+                suffix: (simple_identifier) @param))
+        suffix: (navigation_suffix
+            suffix: (simple_identifier) @access_identifier)) @parameter_access
+(#eq? @param "@stale_flag")
+(#eq? @access_identifier "isEnabled")
+)"""
+replace_node = "parameter_access"
+replace = "false"
```

Comment:
let's replace this with a hole ?  You can pass the value to this hole from `piranha_argument`

Reply:
done